cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -62,20 +70,25 @@ parameter_types! {
 			],
 	);
 	pub storage DeliveryFee: Asset = (Location::parent(), 80_000_000_000u128).into();
+	pub BridgeHub: Location = Location::new(1,[Parachain(westend_runtime_constants::system_parachain::BRIDGE_HUB_ID)]);
 }
 
 impl snowbridge_pallet_system_frontend::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = ();
 	#[cfg(feature = "runtime-benchmarks")]
 	type Helper = ();
-	type CreateAgentOrigin = EnsureXcm<Everything>;
-	type RegisterTokenOrigin = EnsureXcm<Everything>;
+	type CreateAgentOrigin =
+		EitherOf<EnsureXcm<Everything>, EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>>;
+	type RegisterTokenOrigin =
+		EitherOf<EnsureXcm<Everything>, EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>>;
```

Comment:
- Allow user/root origin on AH to create agent
- Allow user/root origin on AH to register token